src/components/organisms/WorldGeneralForm/WorldGeneralForm.tsx

```diff
@@ -117,10 +131,10 @@ export const WorldGeneralForm: React.FC<WorldGeneralFormProps> = ({
 
       reset(omit(input, "creating"));
     },
-    [worldId, user, values, reset, history]
+    [values, user, worldId, reset, history, world?.slug]
   );
 
-  const saveButtonProps: ButtonProps = useMemo(
+  const saveButtonProps: Omit<ButtonProps, "variant"> = useMemo(
```

Comment:
[Q] Will this work?

```suggestion
  const saveButtonProps = useMemo(
```

Reply:
removed `saveButtonProps` entirely. And moved the props directly to the component: https://github.com/sparkletown/sparkle/pull/2868/commits/ba937c968da803ade57ed5627ca35f3baeade6a0